### PR TITLE
Fixed the UnboundLocalError when no issues are assigned to a milestone

### PR DESCRIPTION
**Pull request by goosemania**
The full error was "UnboundLocalError: local variable 'milestone_id'
referenced before assignment".

[noissue]

### CODE REVIEW

File: templates/travis/.travis/release.py.j2 [hunk 1]
```diff
@@ -42,9 +43,12 @@ def validate_redmine_data(redmine_query_url, redmine_issues):
             stats[f"milestone_{milestone}"].append(issue)
         except ResourceAttrError:
             stats["without_milestone"].append(issue)
+        {%- endif %}
 
+    {%- if update_redmine %}
+    if milestone_id is not None:
```
Comment (fao89) [inline templates/travis/.travis/release.py.j2:49]: it is weird, the milestone should be set on line 42

Comment (fao89) [reply templates/travis/.travis/release.py.j2:49]: it should have a milestone_id, otherwise, the job for updating redmine will fail

Comment (goosemania) [reply templates/travis/.travis/release.py.j2:49]: if there are no issues assigned to a milestone, it won't be set.

Comment (goosemania) [reply templates/travis/.travis/release.py.j2:49]: the script will stop early if there is at least one issue not assigned to a milestone, it won't proceed further.

Comment (fao89) [reply templates/travis/.travis/release.py.j2:49]: so we need an if here:
https://github.com/pulp/plugin_template/blob/master/templates/travis/.travis/update_redmine.sh
to check it the commit message contains the redmine query:
https://github.com/pulp/plugin_template/blob/master/templates/travis/.travis/update_redmine.sh

Comment (goosemania) [reply templates/travis/.travis/release.py.j2:49]: What is the case when there is no redmine query and validate method passes so it gets to the update_redmine.sh?
A release with no redmine issues is the only case I can think of.

Comment (fao89) [reply templates/travis/.travis/release.py.j2:49]: I believe this is the only case

**Pull request merged**